app/controllers/exercises_controller.rb

```diff
@@ -47,7 +42,11 @@ def update
       @name = (ExerciseType.find_by(id: @exercise.exercise_type_id)).name
       render plain: "#{@name}"+ " | " + "повторений: " + "#{@exercise.repeats}"+ "," + "подходов: " "#{@exercise.approach}"
     else
-      render plain: "ahuet project"
+      @count = @@count
```

Comment:
@@count - what is it? Why you use it?

Reply:
This variable is used to make divs with unique IDs